ui/lib/apps/SlowQuery/translations/en.yaml

```diff
@@ -34,22 +34,46 @@ slow_query:
     db: Execution Database
     db_tooltip: The database used to execute the query
 
-    parse_time: Parse Time
+    query_time2: Query Time
+    query_time2_tooltip: The execution time of a query (due to the parallel execution, it may be significantly smaller than the following time)
+    parse_time: 　　Parse Time
```

Comment:
Nice trick... I didn't think that. Not sure how it looks like if we don't use mono font.